### PR TITLE
aws-backup: add page

### PR DESCRIPTION
**Pull request by AzadAnsari35**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by blueskyson — APPROVED** [review]

File: pages/common/aws-backup.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# aws backup
+
+> Unified backup service designed to protect Amazon Web Services services and their associated data.
+> More information: <https://docs.aws.amazon.com/cli/latest/reference/backup/index.html>.
+
+- Return BackupPlan details for the specified BackupPlanId:
```
Comment (EmilySeville7cfg) [inline pages/common/aws-backup.md:6]:
```suggestion
- Return BackupPlan details for a specific BackupPlanId:
```

Optional fix: `a specific` may sound better.

File: pages/common/aws-backup.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`aws backup get-backup-plan --backup-plan-id {{id}}`
+
+- Create a backup plan using a backup plan name and backup rules:
```
Comment (EmilySeville7cfg) [inline pages/common/aws-backup.md:10]:
```suggestion
- Create a backup plan using a specific backup plan name and backup rules:
```

The same issue as above.

File: pages/common/aws-backup.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`aws backup create-backup-plan --backup-plan {{plan}}`
+
+- Delete a backup plan:
```
Comment (EmilySeville7cfg) [inline pages/common/aws-backup.md:14]:
```suggestion
- Delete a specific backup plan:
```

The same issue as above.

File: pages/common/aws-backup.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`aws backup delete-backup-plan --backup-plan-id {{id}}`
+
+- Return a list of all active backup plans for an authenticated account:
```
Comment (EmilySeville7cfg) [inline pages/common/aws-backup.md:18]:
```suggestion
- Return a list of all active backup plans for the current account:
```

Optional fix: `the current` may sound better.

File: pages/common/aws-backup.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`aws backup list-backup-plans`
+
+- Return details about your report jobs:
```
Comment (EmilySeville7cfg) [inline pages/common/aws-backup.md:22]:
```suggestion
- List details about your report jobs:
```

Optional fix: `list` may sound better as POSIX-compatible shells don't return anything except exit codes actually.

**Review by EmilySeville7cfg — APPROVED** [review]
- No mandatory to fix issues found, approved.
- This page should be reviewed by `aws backup` users.
- Thanks for a contribution! 🚀

**Pull request merged**